recipes/prinseq-plus-plus/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.2.1" %}
+{% set sha256 = "afd115dbfb3883fcc89b3937801e6b335bcb1e95b5e06c41890713613275ac15" %}
+
+package:
+  name: prinseq-plus-plus
+  version: {{ version }}
+
+source:
+  url: https://github.com/Adrian-Cantu/PRINSEQ-plus-plus/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+  host:
+    - zlib
+    - boost >=1.70.0
```

Comment:
Which one was the right one? `boost`, `boost-cpp`, ...? ping @bgruening

Reply:
Its not clear to me what the difference is?

Reply:
To me neither. It always confuses me, but there are multiple boost packages in conda-forge. Something to do with boost-python I think, and some of it legacy.

Reply:
The naming of those packages is unfortunate, yes.
Basically, at first there was only the `boost` Conda package that included the Python bits, too. But since most of the time people use other parts of Boost, it doesn't make sense to pull in the whole Python stack then (esp. if a package would use one of the non-header-only parts of Boost and have a `run` requirement on `boost`).
To not break existing packages that expected the Python bits to be installed with `boost`, a `boost-cpp` package was created. Unless you want Boost.Python, you should use `boost-cpp`.
(Yes, ideally we'd have `boost`="current `boost-cpp`" and `boost-python`="current `boost`", but this was deemed a breaking change.)